tests/auto/safety_container.test.ts

```
@@ -0,0 +1,418 @@
+
+    it('should handle commands with multiple spaces', async () => {
+      const command = 'echo    "multiple   spaces"';
+      const projectPath = '/test/project';
+      
+      const result = await safetyContainer.runSafeCommand(command, projectPath);
+      
+      expect(result).to.equal('mocked output');
+    });
+
+    it('should handle very long commands', async () => {
+      const command = 'echo "' + 'a'.repeat(1000) + '"';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)